features/app_and_device_attributes.feature

```diff
@@ -53,7 +53,7 @@ Feature: App and Device attributes present
     And the "Bugsnag-API-Key" header equals "12312312312312312312312312312312"
 
     And the payload field "events.0.app.type" equals "iLeet"
-    And the payload field "events.0.app.bundleVersion" does not equal "12345"
+    And the payload field "events.0.app.bundleVersion" equals "12345"
```

Comment:
I don't understand this test - it seems to be setting config values in the client, amending them in a callback but then asserting that the config values are in the payload. I would have expected all these assertions to be against the values set in the callback?

Reply:
Updated E2E test assertions as they were asserting the incorrect behaviour.